docs/chapters/04_features.md

```diff
@@ -487,6 +487,52 @@ mutation {
 
 > [!NOTE]  Remember that, in case you want to subscribe to a read model that is restricted to a specific set of roles, you must send the **access token** retrieved upon sign-in. Check ["Authorizing operations"](#authorizing-operations) to know how to do this.
 
+
+### Validating read models' filters
+You can validate read models by using the `before` hook. The functions you add here will be executed **before fetching from your event-store**:
```

Comment:
Take into account that read models are not stored in the event-store, but in independent tables.
Also, when writing documentation, try to think about the reader and give them clues regarding why the thing you are explaining is useful for them. Be empathic with the reader. Remember the hours you have spent reading bad documentation and how you felt!

```suggestion
When you send queries or subscriptions to your read models, you can tell Booster to execute some code before executing the operation. These are called `before` hooks and they receive two parameters: the read model filter sent with the request and an object with information about the currently signed-in user.
These `before` hooks are useful for many things, but mainly:
- To add fine-grained access control. For example, to allow user A access to its Cart read models, but not the ones belonging to user B.
- Modify the read model filters sent with the request based on some criteria. For example, you can narrow the results the users will get by adding extra filters based on the season of the year.

From inside a `before` hook, you can either throw an exception, in which case the query or subscription will be aborted and the error will be sent back to the user, or return final filters to be used in the operation.

This is an example of how you would define a before hook in a read model: 
```